signer/sign_blinded_beacon_block_test.go

```diff
@@ -107,6 +109,78 @@ func TestBenchmarkBlindedBlockProposalCapella(t *testing.T) {
e14","data":{"slot":"7415519","index":"28","beacon_block_root":"0x4552aa96870d18a017f3fe5bf4a1ff449a4e9e4d5f5300fc0a3624f827208a63","source":{"epoch":"231732","root":"0x1f471edaadf7a2c6ab7d1c0385d45cc83eddff0dede37a827a0062b92a2d2b13"},"target":{"epoch":"231734","root":"0xbf0ea6199d279e3c0efa378a888cf08aa8f5b53eaaf76cae3d539dc538a8fa54"}},"signature":"0x884909ca5b05ed32a75172badcfdb335fe7588ba0f663196e80cebf38b8d352e090600b3381f573f495e5e219799210b028d70a4a4b7e2e4ed0d65a76b2689f500c3dbd518682ee9e6150e8f988f1d04753f4a2ee0eb52dc9316da9d4f5e8ee6"}],"deposits":[],"voluntary_exits":[],"sync_aggregate":{"sync_committee_bits":"0x335fff77bfffb9bff67dff67ffcee77cf3fff679fedfdcfbf2effbef972ebff9b5e1be9fbebaffdde3feef4fbcafecffbeaee3eee9eff8afff3b75bbf77e7df7","sync_committee_signature":"0x923ba7e9f92426f8d3b0206f27ab11bf4d0554286db22c421bd66b4d342404b5c99f6b3fae425c0b652909f131db2db905e5615a103c1c4cb74f1316950fb5a26b751022dce7e9db6f21df74dce008b3158d467ef4c6160f94c5b8818d3fe9fb"},"execution_payload_header":{"parent_hash":"0x1b6059447ea578dad10192b55b3856a25aff1abd942024822a54779784760429","fee_recipient":"0x6de04119406bafbfe88da89267ae7cbe6a4a4bc5","state_root":"0x343fced1f1273752f36555b1959522365d8bebf2de02b6f3aa4e1bf924a7a4f2","receipts_root":"0xe2e5df441ee342a5c3bc4ed0e3bca1114cfd5cf4640abde0bd9a49f58527bfce","logs_bloom":"0x00311810281518010004820481014904002410001810001041014104884659a0521c68020a003020501239c1042c201826d28310981872024090c71256a430120092006a040e004c5000040f802a812f520a414231048095841896049128006842860402df240880031800a02800282319981c006084644018005011400816140824420800000002254036258b001309a0f4d30b103500e80005904015020618460806b43430113850d8484a48080401061b4822510000a06103223a4418060bd201c302110020088117285400144a0024c10400358c401009b00410001c6a80093020080808004d0010000820495202200018002a880040a9081849d2040192","prev_randao":"0xc5647b9f7b3ca7b07b86365095fe47f84ca193c009d91182f9d254b2e9bd59ae","block_number":"10389257","gas_limit":"30000000","gas_used":"8753279","timestamp":"1705494252","extra_data":"0xd883010d0a846765746888676f312e32312e36856c696e7578","base_fee_per_gas":"9","block_hash":"0x202d5d304bf17fec211cd13186fb766dbe610c12574d4c9eadd7e6fdae9ee050","transactions_root":"0x4e2efc021603f6106e875c8d74d5af83103fda595bd5b6cff0f0a4a8935a03d2","withdrawals_root":"0x2457c13873dcd199ffdc4a51f4ae9812d7409418c8e015a7557d8ebceb110fe7","blob_gas_used":"655360","excess_blob_gas":"86507520"},"bls_to_execution_changes":[],"blob_kzg_commitments":["0xc00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000","0xb24758fbdea9852ae96086be16e5634c18a51acc31c9b2b97f7e330bb41065987f73376073c3e1d28f11ccb2d2fc03c2","0xa45a58d23333471a49032949ac925593f2958977c000b0da114092e79910981550fbca5f57c9d9c9aa1e8a90ac0232cf","0x852b25e16fef449d320869f8984f6bda8a04f15a9e6d6cb0fc35c16c8c9e03c377bc387bbca75af799d739cd229672af","0x99943f758bb009192d7dce3d6172e9d466367ac717c6ecab30bd5cb98badf8c1290a8a0529d56f3812f97504def2d970"]}}`)
```

Comment:
where is this taken from?

Reply:
there is a comment above the func testing this block